packages/events/src/service/files/index.ts

```diff
@@ -98,7 +99,9 @@ export async function presignFilesInEvent(
     throw new Error('Failed to find configuration for event')
   }
 
-  const actionFileFields = event.actions.flatMap((action) =>
+  const activeActions = getAcceptedActions(event)
```

Comment:
Definitely not something that you need to action on:

It would be cool to have the "default" event object only include accepted actions and then using a "lense" of sorts on top of it if the intermediate actions also want to be seem. Would potentially reduce dev mistakes